yt_dlp/extractor/loom.py

```diff
@@ -0,0 +1,62 @@
+from .common import InfoExtractor
+from datetime import datetime
+from yt_dlp.utils.traversal import traverse_obj
```

Comment:
use relative imports
```suggestion
from ..utils.traversal import traverse_obj
```